schema/project-level/project-schema.json

```diff
@@ -222,7 +222,7 @@
         },
         "nature": {
           "title": "Nature",
-          "description": "This array field should record whether this contract is related to design, construction or supervision. More than one value may be provided in cases where the contract is for design and build, or design and supervision etc.",
+          "description": "One or more values from the [contractNature](http://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/projects/reference/#contractNature) codelist, indicating whether this contract is related to design, construction or supervision. More than one value may be provided in cases where the contract is for design and build, or design and supervision etc.",
```

Comment:
`
Whether this contracting process relates to the design, construction and/or supervision of the project, from the [contractNature](http://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/projects/reference/#contractnature) codelist. More than one value may be provided if the contract is for both design and construction, or both design and supervision, etc.
`

Lead with the semantics instead of the implementation, as with other codelist fields. Re-use phrasing from code descriptions.